src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -594,7 +567,7 @@ exprt java_string_library_preprocesst::allocate_fresh_array(
 /// \param type: return type of the function
 /// \param symbol_table: a symbol table
 /// \return a function application representing: `function_name(arguments)`
-exprt java_string_library_preprocesst::make_function_application(
+exprt make_function_application(
```

Comment:
static? Also applies to other functions that used to be class members but aren't any longer

Reply:
This function can't be made static, as it's being used in `java_bytecode_typecheck_expr`